### DISCUSSION TITLE
a tick-80 wii port would be intresting

### DISCUSSION THREAD

**Discussion by guinea7pig** [nesbox/TIC-80#2500] [Ideas]
I'm a Wii homebrew enthusiast. a Wii port of tick-80 would be interesting.  it would probably run fine one the Wii, and you would think about editing code on it(though i believe honestly the wiimote for the mouse and a USB keyboard would work fine) probs need permission before  i work on a version if i rewriting the original  code. if there an easier way please message me

**Comment by Miguel-hrvs** [comment]
Here you have people porting some games with devkitpro + sdl2: https://gbatemp.net/threads/sdl-2-port-ongoing.645576/
